tests/transforms/test_broadcast_expand.py

```diff
@@ -52,33 +38,51 @@ def make_tape(x, y, z, obs):
     return tape
 
 
-parameters_and_size = [
-    [(0.2, np.array([0.1, 0.8, 2.1]), -1.5), 3],
-    [(0.2, np.array([0.1]), np.array([-0.3])), 1],
-    [
-        (
-            0.2,
-            pnp.array([0.1, 0.3], requires_grad=True),
-            pnp.array([-0.3, 2.1], requires_grad=False),
-        ),
-        2,
-    ],
+parameters = [
+    (0.2, np.array([0.1, 0.8, 2.1]), -1.5),
+    (0.2, np.array([0.1]), np.array([-0.3])),
+    (
+        0.2,
+        pnp.array([0.1, 0.3], requires_grad=True),
+        pnp.array([-0.3, 2.1], requires_grad=False),
+    ),
 ]
+sizes = [3, 1, 2]
```

Comment:
Separated the two because only one test actually needs the `sizes` but many require the `parameters`